30412/Mosolea Victor Andrei/Extra1/main.c

```diff
@@ -0,0 +1,145 @@
+#include <stdio.h>
+#include "head.h"
+#include <string.h>
+
+int main() {
```

Comment:
Oh gosh, this function almost never ends

Reply:
Please split the main function into some many more smaller ones, I'll merge afterwards